common/data_refinery_common/models/jobs.py

```diff
@@ -87,6 +87,17 @@ class ProcessorJob(models.Model):
     class Meta:
         db_table = "processor_jobs"
 
+        indexes = [
+            models.Index(
+                fields=['created_at'], 
+                name='processor_jobs_created_at',
+                # A partial index might be better here, given our queries we don't
+                # need to index the whole table. We need to update to Django 2.2
+                # for this to be supported.
+                # condition=Q(success=None, retried=False, no_retry=False)
```

Comment:
While this comment seems to have good information, it's likely to be forgotten about here. Do we have an issue open about this? Maybe we can make one and link to it here?

Reply:
Good idea, added here https://github.com/AlexsLemonade/refinebio/issues/1454

Reply:
Good idea, added here #1454

Reply:
Wait can you link to it here?

Reply:
you mean on the comment? sure!

```suggestion
                # condition=Q(success=None, retried=False, no_retry=False)
                # https://github.com/AlexsLemonade/refinebio/issues/1454
```